Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -129,20 +129,10 @@ import { IAuthenticationService } from "@/services/interfaces";
     },
 })
 export default class ReportsView extends Vue {
-    private authenticationService: IAuthenticationService = container.get(
```

Comment:
This line was causing the loggin issue

Reply:
wow nice found, tricky one. Mbe our best practice is to initialize these in the mounted method.